docs/detections/rules-coverage.asciidoc

```diff
@@ -18,25 +18,29 @@ image::images/rules-coverage.png[MITRE ATT&CK® coverage page]
 [discrete]
 === Filter rules
 
-Use the filters at the top of the page to control which of your installed detection rules are included in calculating coverage.
+Use the drop-down filters at the top of the page to control which of your installed detection rules are included in calculating coverage.
 
 * **Installed rule status**: Select to include **Enabled rules**, **Disabled rules**, or both.
 
 * **Installed rule type**: Select to include **Elastic rules** (prebuilt rules), **Custom rules** (user-created rules), or both.
 
+You can also search for a tactic or technique name, technique number, or rule name in the search bar. Only rules matching the search term will be considered for rules coverage.
```

Comment:
This search bar acts as filter for the coverage grid.

Reply:
```suggestion
You can also search for a tactic or technique name, technique number, or rule name in the search bar. The search bar acts as a filter for the coverage grid: only rules matching the search term will be included.
```